src/rabbitmq/rabbitmq.ts

```diff
@@ -33,7 +43,38 @@ export const rabbitConnect = (): void => {
           return connection.close();
         }
         ch.assertExchange(exchange, 'topic', { durable: false });
-        channel = ch;
+
+        ch.assertQueue(
+          '',
+          {
+            exclusive: true
+          },
+          (err2, q) => {
```

Comment:
Why the cryptic names?

Reply:
I'm sorry I need to reevaluate my life decisions here